src/coreclr/jit/fgdiagnostic.cpp

```diff
@@ -1945,12 +1935,7 @@ static volatile int bbTraverseLabel = 1;
 
 void Compiler::fgDebugCheckBBlist(bool checkBBNum /* = false */, bool checkBBRefs /* = true  */)
 {
-#ifdef DEBUG
-    if (verbose)
-    {
-        printf("*************** In fgDebugCheckBBlist\n");
-    }
-#endif // DEBUG
+    JITDUMP("*************** In fgDebugCheckBBlist\n");
```

Comment:
This is generally a fine conversion, to reduce code size and visible ifdefs. However, there is a trade-off: having the "printf" line allows for an easy place to put a breakpoint that only triggers on functions where you've set `COMPlus_JitDump` (so `verbose` is true), so I prefer to leave the original form here for "starting a 'phase'" kind of output like this.

Reply:
That is something I did not consider. Reverted.